File: OCH_Tao/ABC/A/263.js
const L = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ").map(x => parseInt(x, 10));
L.sort((a, b) => a - b);
if ((L[0] === L[2] && L[3] === L[4]) || (L[0] === L[1] && L[2] === L[4])) {
  console.log("Yes");
} else {
  console.log("No");
}

File: OCH_Tao/ABC/A/263.cpp
#include <bits/stdc++.h>
using namespace std;
int main() {
  vector<int> L(5);
  for (int i = 0; i < 5; i++) {
    cin >> L[i];
  }
  sort(L.begin(), L.end());
  cout << ((L[0] == L[1] && L[2] == L[4]) || (L[0] == L[2] && L[3] == L[4]) ? "Yes" : "No") << endl;
}